run-build-functions.sh

```diff
@@ -704,8 +732,3 @@ report_lingering_procs() {
     echo -e "${NC}"
   fi
 }
-
-after_build_steps() {
-  # Find lingering processes after the build finished and report it to the user
-  report_lingering_procs
```

Comment:
we are no longer reporting on lingering_procs?

Reply:
We do, but I just moved it into the run-build scripts.  It's not needed for local builds.